components/results/ResultsFailure.tsx

```diff
@@ -1,14 +1,41 @@
+import ValidationFailure from '../../types/ValidationFailure';
 import CenteredCardContent from '../common/CenteredCardContent';
 
 import ResultsRetryButton from './ResultsRetryButton';
 
-export default function ResultsFailure(): JSX.Element {
+export default function ResultsFailure({ validationFailureDetails }: { validationFailureDetails: ValidationFailure[] }): JSX.Element {
+  // More info about validation: https://documentation.onfido.com/v3.1/#breakdowns
+  const validationMessages = new Map([
+    [ValidationFailure.InvalidImageIntegritySupportedDocument, 'The document provided is not supported'],
+    [ValidationFailure.InvalidImageIntegrityImageQuality, 'The quality of the pictures you provided are too low'],
```

Comment:
I might change 'The quality of the pictures you provided are too low' to something like:
'The picture(s) you provided have quality issues, which may include: blurriness, darkness, glare, obstruction, etc.'
And a comment in the code could point to https://documentation.onfido.com/#image-quality-reasons

Maybe each of these validation errors in this file should have comments pointing to Onfido docs.